crates/typst-library/src/layout/raster.rs

```diff
@@ -0,0 +1,1788 @@
+//! Unified handling for tables and grids.
```

Comment:
To be honest, I'm not sure if I feel comfortable with all the "raster" naming. It is evocative of a series of pixels and not cells, and it's not immediately clear what it represents. While I agree the previous name of `CellGrid` wasn't the best, it did include the fact that it's essentially just a grid. We're basically taking the grid element and transforming it into a more definitive form. And for tables, it's the same thing, as tables are laid out as grids, so it makes sense that they are turned into one.

So I think something like `ResolvedGrid` would be better, and also more consistent with the relevant structures around it. We'd then create a folder `grid/{mod.rs, resolve.rs}`, and not perform significant changes elsewhere. I think this makes for a cleaner architecture. Let me know your thoughts.

Reply:
I see your point about "raster". In my new PR, I have focused on making as few changes as possible, so I kept the existing `CellGrid` name and added `cellgrid` as a new module, leaving the existing `layout/grid.rs` alone.